content/blog/2020/08/2020-08-24-custom-distribution-service.adoc

```diff
@@ -181,6 +181,10 @@ We also included a user docs guide so that it makes it super easy to get started
 
 * Pull Request link:https://github.com/jenkinsci/custom-distribution-service/pull/145[#145]
 
+== Hosting
+
+The project has now been hosted link:https://customize.jenkins.io/[here].
```

Comment:
Would it make sense to mention it is an Alpha version that still needs fixes to be functional?